### ISSUE TITLE
Pressing 'Play' and 'Open Code file' by Enter key does not work

### ISSUE THREAD

**Issue by Helen09** [microsoft/pxt-ev3#541]
**Repro steps:**
• Open https://makecode.mindstorms.com/beta#editor
• Right click on a block, select 'Help'
• Navigate by Tab key on Help page 
• Select 'Play' or 'Open Code File' button by Enter key

**Expected result:**
'Play' or 'Open Code File' button should work normally by Enter key.
**Actual result:**
Pressing 'Play' and 'Open Code file'  by Enter key does not work
![issue3](https://user-images.githubusercontent.com/23466737/39177903-f9c0c7a2-479f-11e8-84d3-599c49b80675.gif)
